Description: Replace Debian logo atop the cube, with LinuxMint's
Author: Jasmine Hassan <jasmine.aura@gmail.com>
Origin: <other>, <URL, required except if Author is present>
Last-Update: 2012-10-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/metadata/cube.xml.in
+++ b/metadata/cube.xml.in
@@ -109,7 +109,7 @@
 			<_long>List of PNG and SVG files that should be rendered on top face of cube</_long>
 			<type>string</type>
 			<hints>file;image;</hints>
-			<default><value>/usr/share/images/desktop-base/debian-blueish-wallpaper-640x480.png</value></default>
+			<default><value>/usr/lib/linuxmint/mintMenu/mint-logo.svg</value></default>
 		    </option>
 		    <option name="adjust_image" type="bool">
 			<_short>Adjust Image</_short>
